code/modules/surgery/cavity_implant.dm

```diff
@@ -154,13 +152,12 @@
 			user.drop_item()
 			affected.hidden = tool
 			tool.forceMove(affected)
-			return TRUE
 	else
 		if(IC)
 			user.visible_message("[user] pulls [IC] out of [target]'s [target_zone]!", "<span class='notice'>You pull [IC] out of [target]'s [target_zone].</span>")
 			user.put_in_hands(IC)
 			affected.hidden = null
-			return FALSE
```

Comment:
In the old system it returned FALSE to stop the stage from advancing. Not required now. This is a total success. Same for the others that used to return FALSE below